deploy/static/06-starboard-operator.deployment.yaml

```diff
@@ -43,13 +43,15 @@ spec:
             - name: OPERATOR_LOG_DEV_MODE
               value: "false"
             - name: OPERATOR_CONCURRENT_SCAN_JOBS_LIMIT
-              value: "3"
+              value: "10"
```

Comment:
We're running 3 different scanners now, so I think it's worth bumping up the default limit